static/app/views/replays/detail/issueList.tsx

```diff
@@ -156,9 +157,12 @@ function IssueList(props: Props) {
       limit={15}
     >
       {data => {
-        return (
+        return data.tableData?.data.length === 0 ? (
+          <Placeholder height="200px">
+            <EmptyMessage title={`${t('No issues for replay event.')}`} />
+          </Placeholder>
+        ) : (
```

Comment:
Ahh, I forgot that the loading and empty states are handled by `PanelTable`.... maybe we just leave this as-is instead of having custom states. Thoughts?